e2e/ansible/playbooks/utils/regex_task.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Replace volume-claim name with test parameters
+  replace:
+    path: {{ path }}
+    regexp: '{{ item.0 }}'
+    replace: '{{ item.1 }}'
+  delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+  with_together:
+    - "{{replace_item}}"
+    - "{{replace_with}}"
+
```

Comment:
We may probably need to test this w/ complex replace with necessity to use escape characters etc.,

Reply:
ok